autotools.bzl

```diff
@@ -0,0 +1,80 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Originally from google/sandboxed-api; added 'bootstrap'.
+
+"""Repository rule that runs autotools' configure after extraction."""
+
+def _configure(ctx):
```

Comment:
As Souffle itself isn't in internal third_party, you'll need to write a BUILD file for that anyway.

Reply:
This is another thing that's directly from sandboxed-api. Since Souffle hasn't been imported yet, I'm planning to defer massaging the build rules until I absolutely have to (or until buildbot gets upset).

Reply:
As nothing in the main repository yet depends on these targets, I'm not sure buildbot will even attempt fetch these, let alone build them.

Reply:
I've manually built the target (on Darwin) and successfully linked an executable that depended on it (I instantiated an arbitrary utility class to make sure the library wasn't completely discarded).